shanoir-ng-front/src/app/async-tasks/task.model.ts

```diff
@@ -95,15 +95,15 @@ export class Task extends Entity {
             } else if (this.message.indexOf('dataset [') != -1) {
                 let substring = this.message.match(/dataset \[\d+\]/g)[0];
                 return '/dataset/details/' + substring.slice(substring.lastIndexOf("[") + 1, substring.lastIndexOf("]"));
-            } else if (this.message.indexOf('VIP Execution') != -1) {
-               return '/dataset-processing/details/' + this.objectId
             }
+        } else if (this.eventType === 'executionMonitoring.event' && this.status != -1) {
```

Comment:
=== is not used in java, and I think you can create an event type in ShanoirEventType.java to have the reference. (event status is in a constant on ShanoirEvent I think too)